use_cases/index.md

```diff
@@ -7,13 +7,9 @@ custom_repo_branch: datahub-latest-documentation
 ---
 See for yourself how DataHub is being used or where it has been integrated in.
```

Comment:
please, replace it with " Explore both existing and upcoming applications and integrations of DataHub. "